profiles/ucentral-ap.yml

```diff
@@ -9,7 +9,6 @@ feeds:
 include:
   #- webui
   - openflow
-  - fbwifi
```

Comment:
NIT, but can we comment it out, like we did for webui?

Reply:
fine. I will do it.

Reply:
more NIT.  #fbwifi should replace fbwifi.  It should be on the same line it was originally

Reply:
@bmastbergen Done please review.